src/messages.ts

```diff
@@ -11,7 +11,6 @@ export const help = function() {
   return `
     Only ${chalk.green('<project-directory>')} is required.
     If you have any problems, do not hesitate to file an issue:
-      ${chalk.cyan('https://github.com/formik/tsdx/issues/new')}
```

Comment:
think you might have accidentally made a mistake here as it got deleted instead of updated

Reply:
sorry, this was an oversight. fixed now.